util/binwarp/binwarp.lisp

```diff
@@ -0,0 +1,105 @@
+;;;; binwarp.lisp
+
+(in-package #:binwarp)
+
+(declaim (special *binwarp-mode-p* *binwarp-area* *binwarp-history*
+                  *reinitiate-ptr* *init-ptr-position*
+                  *preserve-history*))
```

Comment:
```suggestion
```

You do no't need to declaim the variables as special for variables that are defined in the same file.